en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/endpoints/defining-a-backend-security-scheme.md

```diff
@@ -2,73 +2,47 @@
-
-**Define and refer endpoints**
-
-``` yaml
-x-wso2-basePath: /petstore/v1
-x-wso2-production-endpoints: "#/x-wso2-endpoints/myEndpoint"
-
-paths:
-    "/pet/findByStatus":
-       get:
-       tags:
-         - pet
-       summary: Finds Pets by status
-       description: Multiple status values can be provided with comma separated strings
-       operationId: findPetsByStatus
-       x-wso2-production-endpoints: "#/x-wso2-endpoints/myEndpoint3"
-    
-x-wso2-endpoints:
-  - myEndpoint:
-    urls:
-    - https://petstore.swagger.io/v2
-    - https://petstore.swagger.io/v5
-  securityConfig:
-    type: basic
-    username: roshan
-  - myEndpoint3:
-    urls:
-    - https://petstore.swagger.io/v3
-    - https://petstore.swagger.io/v4
-  securityConfig:
-    type: basic
-    username: kim
-```
-
-A complete sample can be found [here.](https://github.com/wso2/product-microgateway/blob/master/samples/endpoint_by_reference_sample.yaml)
-
-##### Invoking an API using basic authentication
+When an actual backend service of the API is protected using basic authentication, the basic authentication parameters (username and password) must be sent to the backend. Hence, you need to define the endpoint security parameter in the api.yaml as follows.
 
-When running the micro gateway, you can provide the password as an environment variable.
+  ```
+  endpointConfig:
+    endpoint_security
+      production:
+        type: "BASIC"
+        username: "admin"
+        password: "admin"
+      sandbox:
+        type: "BASIC"
+        username: "admin"
+        password: "admin"
+  ```
+## Invoking an API using basic authentication
 
-``` text tab="Format"
-bash gateway <path_to_the_jar_file> --<Endpoint Name>_<Endpoint Type>_basic_username=<username> --<Endpoint Name>_<Endpoint Type>_basic_password=<password> 
+When running the Choreo Connect, you can provide the username, password as adapter environment variables at docker-composer.yaml file.
 
-- Endpoint Name: Name specified in the open API definition under `x-wso2-endpoints`
-- Endpoint Type: prod or sand.
-```
+!!! note 
+    If a username/password is not specified as environment variables, the username, password defined in the API Definition section will be used while deploying the 
+    API to Choreo Connect.
 
-``` text tab="Example - binary"
-bash gateway petstore.jar --myEndpoint3_prod_basic_username=admin --myEndpoint3_prod_basic_password=123456
 ```
-``` text tab="Example - docker"
-docker run -d -p 9090:9090 -p 9095:9095 -e myEndpoint3_prod_basic_username=admin -e myEndpoint3_prod_basic_password=123456 docker.wso2.com/petstore:v1
+<API-ID>_<endpoint-type>_basic_username=<username>
+<API-ID>_<endpoint-type>_basic_password=<password>
 ```
 
-!!! note 
-    If a username is not specified, the username defined in the API Definition section while implementing the API, will be used.
+- API-ID : `api_SHA1(<apiName>:<apiVersion>:<basepath>)`
```

Comment:
Did we included the base path, also, I though it was only name:version combination

Reply:
My bad. it should be vhost. thanks for pointing it.

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/3568/commits/0bcdd999288f332d0bd6308bff36e8f7648f4e8c